src/Sylius/Bundle/CoreBundle/Form/Extension/LocaleTypeExtension.php

```diff
@@ -66,9 +66,6 @@ private function getLocaleName(string $code): ?string
         return Locales::getName($code);
     }
 
-    /**
-     * @return array|LocaleInterface[]
-     */
```

Comment:
Why did you remove `LocaleInterface[]` type-hint?

Reply:
It was not right, the method returns a string[]. It could be automatically recognized by PHPStan but I anyway added it to the doc.